app/imports/templates/status/open.html

```diff
@@ -10,18 +10,20 @@ <h1 class="domain-name"><strong>{{name}}</strong> is available</h1>
 
         {{#if hasNode}}
   
-          <p><input type="checkbox" id="agreement" name="agreement"> <label for="agreement">I understand <a href="#terms" class="explainer">how the process works and my risks and responsibilities involved </a>.</label></p>
-          {{#if TemplateVar.get 'agree'}}
-          <button type="submit" class="dapp-block-button open-auction" disabled={{opening}}> 
+          {{>components_newBid}}
+
+          <p>
+          Or..
+          </p>
+
+          <button type="submit" class="open-auction" disabled={{opening}}> 
             {{#if opening}}
               Opening...
             {{else}}
-              Open auction
+              Just open, don't bid now
```

Comment:
Oh, now I understand the text; I thought it was informational ("It was just opened, don't bid now"). Perhaps "Open without bidding" instead?